packages/snaps-utils/src/manifest/manifest.ts

```diff
@@ -393,13 +393,15 @@ export function getWritableManifest(manifest: SnapManifest): SnapManifest {
  * @param snapFiles.sourceCode - The Snap's source code.
  * @param snapFiles.svgIcon - The Snap's optional icon.
  * @param snapFiles.auxiliaryFiles - Any auxiliary files required by the snap at runtime.
+ * @param snapFiles.localizationFiles - The Snap's localization files.
  */
 export function validateNpmSnapManifest({
```

Comment:
I think you are missing some changes to `checkManifest` here 🤔

Reply:
What do you mean?